src/components/_overlays/MemberDropdown/MemberDropdown.styles.ts

```diff
@@ -9,27 +10,38 @@ const paddingStyles = css`
   padding: ${sizes(6)} ${sizes(4)};
 `
 
-export const Container = styled.div<{ isActive: boolean }>`
+export const Container = styled.div`
   position: fixed;
   right: ${sizes(4)};
   top: 0;
   width: 280px;
-  max-height: calc(100vh - ${sizes(4)} - var(--size-topbar-height));
-  overflow-x: hidden;
-  overflow-y: auto;
-  background-color: ${cVar('colorCoreNeutral700')};
-
-  transform: translateY(${({ isActive }) => (isActive ? 'var(--size-topbar-height)' : '-100%')});
-  transition: transform ${transitions.timings.loading} ${transitions.easing};
+  height: 0;
   z-index: ${zIndex.nearTransactionBar};
 
-  box-shadow: ${cVar('effectElevation24Layer2')}, ${cVar('effectElevation24Layer1')};
-
   ${media.md} {
     right: ${sizes(8)};
   }
 `
 
+export const InnerContainer = styled.div<{ isActive: boolean; containerHeight: number }>`
+  width: 280px;
+  position: 'relative';
+  max-height: calc(100vh - ${sizes(4)} - var(--size-topbar-height));
+
+  height: ${({ containerHeight }) => containerHeight}px;
+  transform: translateY(
+    ${({ isActive, containerHeight }) => (isActive ? 'var(--size-topbar-height)' : -containerHeight + 'px')}
+  );
+  transition: transform ${transitions.timings.loading} ${transitions.easing},
+    height ${cVar('animationTimingMedium')} ${cVar('animationEasingMedium')};
```

Comment:
You can just write: 
```suggestion
    height ${cVar('animationTransitionMedium')};
```
Also, I think we should avoid using `{transitions.timings.loading} ${transitions.easing},` it should be deprecated soon. Could we change it to `animationTransitionSlow` ? Is there a reason that we used these values here?